ios/kmbuild.sh

```diff
@@ -166,14 +171,16 @@ echo "Build products will be set with the following version metadata:"
 echo "  * VERSION=$VERSION"
 echo "  * VERSION_WITH_TAG=$VERSION_WITH_TAG"
 echo "  * VERSION_ENVIRONMENT=$VERSION_ENVIRONMENT"
+echo "  * DO_SENTRY=$UPLOAD_SENTRY"
```

Comment:
Why the change of variable name? It seems unnecessary and a hassle for future maintainers?

Reply:
Sadly, because I went with `DO_SENTRY` first before realizing it should mirror the actual build flag... after all the Xcode project work was already done.  Hence the reluctance to fully commit to the better name.  I'll get it fixed up.